docs/toolsets.md

```diff
@@ -708,3 +708,42 @@ toolset = ACIToolset(
 
 agent = Agent('openai:gpt-5', toolsets=[toolset])
 ```
+
+### Pydantic-Deep {#pydantic-deep}
+
+[Pydantic-Deep](https://github.com/vstorm-co/pydantic-deepagents) implements the [Deep Agents](https://blog.langchain.com/deep-agents/) architecture pattern for Pydantic AI. Deep agents are production-grade agents with planning, file system operations, task delegation, and sandboxed code execution — patterns documented from production systems like Claude Code and Manus.
+
+Available toolsets:
+
+- `TodoToolset`: Task planning and progress tracking for agent self-organization
+- `FilesystemToolset`: File operations (ls, read, write, edit, glob, grep, execute)
+- `SubAgentToolset`: Spawn and delegate tasks to specialized subagents
+- `SkillsToolset`: Load and use [agent skills](https://www.anthropic.com/engineering/equipping-agents-for-the-real-world-with-agent-skills) from markdown files
+
+The framework also includes multiple backends (in-memory, filesystem, Docker sandbox), automatic conversation summarization, and human-in-the-loop approval workflows.
+
+You will need to install the `pydantic-deep` package.
+
+```python {test="skip" lint="skip"}
```

Comment:
No code sample please, it'll just get out of date, so I'd prefer to link to your docs only